modules/cli-manager-adding-plugin-yaml.adoc

```diff
@@ -0,0 +1,65 @@
+// Module included in the following assemblies:
+//
+// * cli_reference/cli_manager/cli-manager-using.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="cli-manager-adding-plugin-yamls_{context}"]
+= Adding plugins to the {cli-manager} with YAML files
+
+.Prerequisites
+
+* The {cli-manager} is installed.
+* You are using {product-title} 4.17 or later.
+* You are logged in to {product-title} as a user with the `cluster-admin` role.
+
+.Procedure
+
+. Log in to the {product-title} web console.
+
+. Navigate to *Operators* -> *Installed Operators*.
+
+. Select the *{cli-manager}* from the list.
+
+. Select the *CLI Plugin* tab.
+
+. Click *Create Plugin*.
+
+. Enter the YAML file with the information for the plugin you are installing in the text box. See the example YAML file below.
++
+.Example YAML file to add a plugin
+[source,yaml]
+----
+apiVersion: config.openshift.io/v1alpha1
+kind: Plugin
+metadata:
+  name: <plugin_name>
+spec:
+  shortDescription: short description of the <plugin_name>
+  description: long description of the <plugin_name>
+  version: <version> <1>
+  caveats: <2>
+  homepage: <plugin_homepage>
+  platforms:
+  - platform: <platform> <3>
+    image: <plugin_image>
+    imagePullSecret: <4>
+    files:
+    - from: <plugin_file_path>
+----
+<1> Version must be in v0.0.0 format.
+<2> This field is optional.
+<3> Linux/amd64 # Linux/arm64, darwin/arm64, windows/amd64, or another.
```

Comment:
```suggestion
<3> linux/amd64 # linux/arm64, darwin/arm64, windows/amd64, or another.
```

Reply:
I'm not sure we should mention this or not.

We made a change in https://issues.redhat.com/browse/OCPBUGS-43549. For darwin and windows compatible binaries, cli-manager will extract them from linux/amd64 image (if the image is manifestlist). Because there is no darwin or windows base images anyway.

WDYT @kasturinarra ?

Reply:
@ardaguclu  we can mention once i finish testing may be ? Also could we check with gaurav as well to see what he thinks ?

Reply:
> We made a change in https://issues.redhat.com/browse/OCPBUGS-43549. For darwin and windows compatible binaries, cli-manager will extract them from linux/amd64 image (if the image is manifestlist). Because there is no darwin or windows base images anyway.

@cbippley please ignore this comment as well.